bugsnag-android-core/src/main/java/com/bugsnag/android/Stackframe.kt

```diff
@@ -106,10 +106,10 @@ class Stackframe : JsonStream.Streamable {
         lineNumber = json["lineNumber"] as? Number
         inProject = json["inProject"] as? Boolean
         columnNumber = json["columnNumber"] as? Number
-        frameAddress = (json["frameAddress"] as? Number)?.toLong()
-        symbolAddress = (json["symbolAddress"] as? Number)?.toLong()
-        loadAddress = (json["loadAddress"] as? Number)?.toLong()
-        codeIdentifier = (json["codeIdentifier"] as? String)
+        frameAddress = (json["frameAddress"] as? String)?.let { java.lang.Long.decode(it) }
+        symbolAddress = (json["symbolAddress"] as? String)?.let { java.lang.Long.decode(it) }
+        loadAddress = (json["loadAddress"] as? String)?.let { java.lang.Long.decode(it) }
```

Comment:
This will need to handle cases where the String value represents an unsigned high-value long (where the most-significant bit is set). `Long.decode` will throw a `NumberFormatException` in these cases.